man/man1/rtl_433.1

```diff
@@ -610,6 +610,9 @@ Interval Data Message (IDM) for Net Meters
 .TP
 [ \fB163\fI\fP ]
 Acurite 590TX Temperature with optional Humidity
+.TP
+[ \fB164\fI\fP ]
+Secplus v2
```

Comment:
The actual name of the system is "Security+ 2.0". Secplus is just the name of my Python implementation.